### DISCUSSION TITLE
Question: can someone make mod music and vocal for me.

### DISCUSSION THREAD

**Discussion by Mayu-can** [FunkinCrew/Funkin#1530] [Q&A] [answered]
Hallo. 

I have a question. 
Can someone maybe help me by doing the music and vocal part for my mod? I really tried to do it myself but it just doesn't work out. I tryed everything and watch every video that I could find, but no luck. Im just not good at it. 
So if someone can help me and do it for me that would be a amazing.

**Comment by SilverT4** [comment, marked as answer]
Please use something like [GameBanana](https://gamebanana.com/games/8694) for this kind of question.
